library.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "tinyfsm",
+    "version": "0.3.1",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/digint/tinyfsm"
```

Comment:
main repo is at: https://dev.tty0.ch/tinyfsm.git
secondary is github: https://github.com/digint/tinyfsm.git

(notice .git suffix)

Reply:
Git link fixed in latest commit.